src/pages/Registration/Steps/Banking/useSubmit.ts

```diff
@@ -1,43 +1,71 @@
-import { SubmitHandler, UseFormReturn } from "react-hook-form";
+import { useCallback, useState } from "react";
 import { useNavigate } from "react-router-dom";
-import { FormValues as FV } from "./types";
+import { CreateRecipientRequest } from "types/aws";
+import { FileDropzoneAsset } from "types/components";
+import { useCreateRecipientAccountMutation } from "services/aws/bankDetails";
 import { useUpdateRegMutation } from "services/aws/registration";
 import { useErrorContext } from "contexts/ErrorContext";
+import { getFilePreviews } from "helpers";
+import { GENERIC_ERROR_MESSAGE } from "constants/common";
 import { steps } from "../../routes";
 import { useRegState } from "../StepGuard";
 
-export default function useSubmit(form: UseFormReturn<FV>) {
+export default function useSubmit() {
   const { data } = useRegState<5>();
-  const {
-    handleSubmit,
-    formState: { isDirty, isSubmitting },
-  } = form;
-
+  const [createRecipientAccount] = useCreateRecipientAccountMutation();
   const [updateReg] = useUpdateRegMutation();
   const { handleError } = useErrorContext();
   const navigate = useNavigate();
+  const [isSubmitting, setSubmitting] = useState(false);
+
+  const submit = useCallback(
```

Comment:
when i try to submit i get error
```
Request URL:
https://kpnxz5rzo2.execute-api.us-east-1.amazonaws.com/staging/wise
Request Method:
POST
Status Code:
422 Unprocessable Content
Remote Address:
44.205.162.160:443
Referrer Policy:
strict-origin-when-cross-origin
```

payload 
```
{
    "method": "POST",
    "url": "/v1/accounts",
    "PK": "a037a29b-f328-46de-94b0-eceb38e81499",
    "payload": "{
    "method": "POST",
    "url": "/v1/accounts",
    "PK": "a037a29b-f328-46de-94b0-eceb38e81499",
    "payload": "{\"accountHolderName\":\"testorg\",\"currency\":\"PHP\",\"type\":\"philippinesmobile\",\"details\":{\"legalType\":\"PRIVATE\",\"email\":\"mail@mail.com\",\"accountHolderName\":\"test last\",\"bankCode\":\"GCASH\",\"accountNumber\":\"9983651234\",\"address.country\":\"PH\",\"address.city\":\"city\",\"address.firstLine\":\"address1\",\"address.postCode\":\"1111\"}}"
}
```

Reply:
Can you post the exact repro steps please?

Reply:
REPRO in registration banking page
* select `PHP` as currency
* any amount
* choose `e-wallet` 
* recipient account number `9983651234` postcode `1111`, country: Philippines, city: city
* check requirements button
* continue

Reply:
Working on this, will coordinate with Paolo if necessary

Reply:
Solved all the (apparent) errors on webapp side, now getting `400 Bad Request`.
This is most likely due to a missing Profile ID assignment on AWS side, waiting on Paolo to implement this.